schema/crdb/dbinit.sql

```diff
@@ -3955,7 +3951,23 @@ CREATE TABLE IF NOT EXISTS omicron.public.bp_omicron_zone (
     -- Eventually, that nullability should be removed.
     filesystem_pool UUID,
 
-    PRIMARY KEY (blueprint_id, id)
+    -- Zone disposition
+    disposition omicron.public.bp_zone_disposition NOT NULL,
+
+    -- Specific properties of the `expunged` disposition
+    expunged_as_of_generation INT,
```

Comment:
Same as in the rust code, I think it would be handy to also prefix this with `disposition_`.